operator/pkg/central/extensions/reconcile_pvc.go

```diff
@@ -225,15 +255,25 @@ func (r *reconcilePVCExtensionRun) getUniqueOwnedPVCs() (*corev1.PersistentVolum
 		return nil, nil
 	}
 
+	// Filter PVC List by current PVC Claim Name
+	filtered := pvcList[:0]
+	for _, pvc := range pvcList {
+		// If the target annotation is empty, default to Central for backwards compatibility
+		if val, ok := pvc.Annotations[pvcAnnotationKey]; !ok && r.target == PVCTargetCentral {
```

Comment:
Ah! Thanks!

Reply:
Annotations for sure

Reply:
Well, in an ideal world - without backwards compatibility requirements -, this should really be a label, no? Because we _would_ use it for querying objects (and actually we still _can_ use it for querying if `r.target != PVCTargetCentral`)?

Reply:
Yeah, that makes sense to me, but I probably won't split the logic more for the label querying considering the realistically small benefit

Reply:
sorry I will actually do this because I do it for the annotation check so it's functionally the same

Reply:
...and flipping back, will not do it